src/Fable.Import.ReactNative.fs

```diff
@@ -311,58 +311,92 @@ module ReactNative =
         abstract delete: LayoutAnimationAnim option with get, set
 
     and LayoutAnimationStatic =
-        abstract configureNext: Func<LayoutAnimationConfig, Func<unit>, Func<obj, unit>, unit> with get, set
-        abstract create: Func<float, string, string, LayoutAnimationConfig> with get, set
+        abstract configureNext: (LayoutAnimationConfig -> (unit -> unit) -> (obj -> unit) -> unit) with get, set
+        abstract create: (float -> string -> string -> LayoutAnimationConfig) with get, set
         abstract Types: LayoutAnimationTypes with get, set
         abstract Properties: LayoutAnimationProperties with get, set
-        abstract configChecker: Func<obj, string, string, unit> with get, set
+        abstract configChecker: (obj -> string -> string -> unit) with get, set
         abstract Presets: obj with get, set
 
     and [<StringEnum>] FlexAlignType =
-        | ``Flex-start`` | ``Flex-end`` | Center | Stretch
+        | ``Flex-start`` | ``Flex-end`` | Center | Stretch | Baseline
```

Comment:
If we're making so many changes, maybe it's a good time to use `| [<CompiledName("flex-start")>] FlexStart` here (and in the other similar cases) to avoid the need of quotation. This is done for example, with [http properties](https://github.com/fable-compiler/fable-powerpack/blob/6cc582d2f77c84f1c9f645ec4914f5660ed544ae/src/Fetch.fs#L55-L61).